data/human/transport missions.txt

```diff
@@ -3209,3 +3209,65 @@ mission "Earth Retirement"
 			label reject
 			`	Charles looks surprised for a moment, and then, for the first time you've seen, he smiles. "Thank you, Captain <last>. I wish there were more people like you in the Paradise Worlds."`
 			`	You wish the couple the best of luck on Earth, and they slowly make their way to the train station. Presumably, their new apartment is far from here.`
+
+mission "Human to Hai Space"
+	description "Bring a human passenger to <destination> in Hai space."
+	minor
+	source
+		government "Republic" "Syndicate" "Free Worlds"
+		attributes "tourism" "urban"
+	destination 
+		government "Hai"
+		attributes "human tourism"
+	passengers 1
+	to offer
+		random < 10
+	on offer
+		conversation
+			`A young person approaches you carrying luggage. "You look like a knowledgeable captain," he says. "Do you know of a place I can go to visit some... intelligent rodents?"`
+			branch "hai contact"
+				has "First Contact: Hai: declined"
+			
+			choice
+				`   "What are you talking about?"`
+				`   "What, like in a zoo?"`
+					goto "confused"
+				`   "Are you high?"`
+			`   "No, I want to go to the Hai-" He stops once he realizes you have no idea what he's talking about. "Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`
+				decline
+			label "confused"
+			`   "Oh, never mind then," "Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`
+				decline
+			
+			label "hai contact"
+			choice
+				`   "You mean the Hai? You'll have to go up north for that."`
+					goto "squirrel transport"
+				`   "Ah, I think I know of some large squirrels to the north of here."`
+					goto "squirrel transport"
+				`   (Feign ignorance.)`
+			`   "I have no idea what you're talking about," you say. "Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`
+			label "squirrel transport"
+			`   "If that's the case, could you bring me to <destination>? I'll give you <payment>."`
+			choice
+				`   "Alright, I'll take you there."`
+					accept
+				`   "Sorry, I'm too busy right now."`
+					decline
+				`   "Why would you want to go all the way out to <destination>?"`
```

Comment:
`<destination>` is replaced by `<planet> in the <system> system`. Having it in both the question and the answer feels weird:
```
- "If that's the case, could you bring me to Greenwater in the Fah Soom system? I'll give you <payment>."
- "Why would you want to go all the way out to Greenwater in the Fah Soom system?"
```

Using only \<planet\> or \<system\> in the answer would be better:
```
- "If that's the case, could you bring me to Greenwater in the Fah Soom system? I'll give you <payment>."
- "Why would you want to go all the way out to Greenwater?"
```